internal/awsservice/constant.go

```diff
@@ -37,8 +39,14 @@ var (
 )
 
 var (
+	retryer = func() aws.Retryer {
```

Comment:
Since I'm creating so many log streams. When we go to delete log stream we get throttle limits.